src/IbexaPlatformAssetsBundle/bundle/IbexaPlatformAssetsBundle.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Platform\Bundle\Assets;
+
+use Ibexa\Platform\Bundle\Assets\DependencyInjection\Configuration\Parser;
+use Ibexa\Platform\Bundle\Assets\DependencyInjection\IbexaPlatformAssetsExtension;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+final class IbexaPlatformAssetsBundle extends Bundle
+{
+    public function getContainerExtension(): ExtensionInterface
+    {
+        return new IbexaPlatformAssetsExtension();
+    }
+
+    public function build(ContainerBuilder $container)
+    {
+        /** @var \eZ\Bundle\EzPublishCoreBundle\DependencyInjection\EzPublishCoreExtension $kernelExtension */
+        $kernelExtension = $container->getExtension('ezpublish');
```

Comment:
Isn't `ezplatform` a wrapper for `ezpublish` extension? Ping: @alongosz

Reply:
> Isn't `ezplatform` a wrapper for `ezpublish` extension? Ping: @alongosz

Right now it's a separate extension, not inheriting or sharing anything. Config is copied to ezpublish via compiler pass magic. This was the quickest way to deliver it. As a consequence EzPlatformCoreExtension won't have e.g. `addConfigParser` method or won't even properly prepend config. I need to work on that...